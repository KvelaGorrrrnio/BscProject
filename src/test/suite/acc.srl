// Compute the accumulated sum

int n int a

n ^= 10
from (a = 0) do
  a += n
  n -= 1
until (n = 0)
