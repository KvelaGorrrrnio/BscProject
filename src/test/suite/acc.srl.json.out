{ "type" : "variable", "table" : [{ "id" : "a", "value" : "55" }, { "id" : "n", "value" : "0" }] }
